environments/liveness/liveness-environment/amplify/backend/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+typescript@^4.9.5:
```

Comment:
Curious how this snuck in? Don't see any dep changes

Reply:
Yeah it was already there locally just not included in git. I had initially not included because it wasn't included in the previous liveness environment but adding it here to reduce the number of changes people would have to discard after pulling